tests/api/templates.test.js

```diff
@@ -105,4 +112,76 @@ describe("Test JSON validation scenarios", () => {
     expect(res.statusCode).toBe(400);
     expect(JSON.parse(res._getData()).error).toContain('instance requires property "form"');
   });
+
+  it("Should log admin activity if POST API call completed successfully", async () => {
```

Comment:
I think this test can be removed by simply adding `expect(logFormUpload).toHaveBeenCalled()` in the first test. Perhaps you can rename the first test to `Should successfully handle a POST request to create a template`